.concourse/tasks/upgrade.sh

```diff
@@ -108,14 +108,14 @@ make kubecf
 tcp_router_ip=$(kubectl get svc -n scf tcp-router-public -o json | jq -r .status.loadBalancer.ingress[].ip | head -n 1)
 public_router_ip=$(kubectl get svc -n scf router-public -o json | jq -r .status.loadBalancer.ingress[].ip | head -n 1)
 
-gcloud --quiet beta dns --project=${GKE_PROJECT} record-sets transaction start \
-       --zone=${GKE_DNS_ZONE}
-gcloud --quiet beta dns --project=${GKE_PROJECT} record-sets transaction add \
-       --name=\*.${DOMAIN}. --ttl=300 --type=A --zone=${GKE_DNS_ZONE} $public_router_ip
-gcloud --quiet beta dns --project=${GKE_PROJECT} record-sets transaction add \
-       --name=tcp.${DOMAIN}. --ttl=300 --type=A --zone=${GKE_DNS_ZONE} $tcp_router_ip
-gcloud --quiet beta dns --project=${GKE_PROJECT} record-sets transaction execute \
-       --zone=${GKE_DNS_ZONE}
+gcloud --quiet beta dns --project="${GKE_PROJECT}" record-sets transaction start \
+       --zone="${GKE_DNS_ZONE}"
+gcloud --quiet beta dns --project="${GKE_PROJECT}" record-sets transaction add \
+       --name="\*.${DOMAIN}." --ttl=300 --type=A --zone="${GKE_DNS_ZONE}" "$public_router_ip"
+gcloud --quiet beta dns --project="${GKE_PROJECT}" record-sets transaction add \
+       --name=tcp."${DOMAIN}." --ttl=300 --type=A --zone="${GKE_DNS_ZONE}" "$tcp_router_ip"
```

Comment:
```suggestion
       --name="tcp.${DOMAIN}." --ttl=300 --type=A --zone="${GKE_DNS_ZONE}" "${tcp_router_ip}"
```
(Purely personal style, no need to accept if you don't like it)